renovate.json

```diff
@@ -11,6 +11,12 @@
       "groupName": "gitea dependency",
       "groupSlug": "gitea",
       "matchPackageNames": ["gitea/gitea"]
+    }, 
+    {
+      "matchDatasources": ["docker"],
+      "matchPackageNames": ["node"],
+      "versionCompatibility": "^(?<version>^20.*)(?<compatibility>-.*)?$",
```

Comment:
I might see a potential issue with this rule is that Renovate won't perform the upgrade to v21 when v21 becomes the new LTS version 🤔